workspaces/global-header/README.md

```diff
@@ -14,3 +14,23 @@ To generate knip reports for this app, run:
 ```sh
 yarn backstage-repo-tools knip-reports
 ```
+
+To test Global Header with Notifications plugin, you need to set up an external service as described in the [Backstage documentation](https://backstage.io/docs/auth/service-to-service-auth/#static-keys-for-plugin-to-plugin-auth). For example, you can use the following configuration:
+
+```yaml
+backend:
+  auth:
+    externalAccess:
+      - type: static
+        options:
+          token: ${EXTERNAL_SERVICE_TOKEN}
+          subject: cicd-system-completion-events
+```
+
+To send notifications, you can use the following command:
+
+```sh
+curl -X POST http://localhost:7007/api/notifications -H "Content-Type: application/json" -H "Authorization: Bearer ${EXTERNAL_SERVICE_TOKEN}" -d '{"recipients":{"type":"broadcast"},"payload": {"title": "Title of boradcast message","link": "http://foo.com/bar","severity": "high","topic": "The topic"}}'
```

Comment:
Thanks for this! :)

I read it first in the official doc and they said the bearer token should be base64 encoded. Looks like that isn't true.

Btw, I couldn't start the workspace without adding this.

You can maybe just change the default to `token: EXTERNAL_SERVICE_TOKEN`.

For this service token I think we can live with an "hard coded" token. Wdyt?

Reply:
Hi @christoph-jerolimov , the `${EXTERNAL_SERVICE_TOKEN}` is an expression for environment variables. All env variables in our existing yaml files are in this format. Better we don't use a new expression only for this file, it could be confusing.

Instead of a hardcoded value, I can attach the link to the Backstage document where explains how to generate an `EXTERNAL_SERVICE_TOKEN` value. The reason I want to keep this name is that it shows exactly what it is. Otherwise a string could be confusing, like what is that and where is that from. What do you think?

Reply:
Of course it must not be called external_service_token then. This might confuse people.

I hope this is fine for you: https://github.com/redhat-developer/rhdh-plugins/pull/373/commits/f9731fe401d44f845951c322123491f9c6995301

If not please really let me know. Then I'm sorry. :)